ci-operator/config/openshift-kni/numaresources-operator/openshift-kni-numaresources-operator-main.yaml

```diff
@@ -3,6 +3,10 @@ base_images:
     name: cli-operator-sdk
     namespace: ocp
     tag: v1.31.0
+  operator-sdk:
```

Comment:
I see it's just a name but shouldn't this be 4.18? maybe it is consumed somewhere important

Reply:
The only image it found was 4.17